soc/riscv/riscv-privileged/hpmicro/Kconfig.soc

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 HPMicro
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+prompt "HPMicro HPM6700_6400 SoC Selection"
+depends on SOC_SERIES_HPMICRO
+
+config SOC_HPM6750
+	bool "HPMicro HPM6750 SoC implementation"
+	select ATOMIC_OPERATIONS_BUILTIN
+	select INCLUDE_RESET_VECTOR
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
```

Comment:
According to the user manual description HPM6750 support:
RV32I, M, A, F, D, C, P(SIMD and DSP)
so you have lost:
RISCV_ISA_EXT_F
RISCV_ISA_EXT_D
RISCV_ISA_EXT_C